colcon_core/task/python/build.py

```diff
@@ -244,10 +249,14 @@ def _symlinks_in_build(self, args, setup_py_data):
                 os.path.join(args.build_base, item)))
         # provide a symlink within the build space if a module name is
         # changed by the mapping specified in package_dir
+        temp_symlinks = []
         for rel_src, rel_dst in renamed_items:
             symlinks.append((
-                os.path.join(args.build_base, rel_src),
+                os.path.join(args.path, item),
```

Comment:
`item` seems wrong here since it is not set in this loop but only by earlier code.

Reply:
Oops, `item` -> `rel_src` in d425ef1. This time I tested locally with a package that has two renamed items.